.pre-commit-config.yaml

```diff
@@ -27,7 +27,7 @@ repos:
 # The structure of this was suggested by the author of pre-commit and maintainer of flake8
 # See https://stackoverflow.com/a/66485642/3956024
 - repo: https://github.com/PyCQA/flake8
-  rev: 4.0.1
+  rev: 5.0.2
```

Comment:
Should we then also update the flake8 version in the `environment.yml` files?

Reply:
Good point, yes. I would just removed the version from that requirement, actually.

Reply:
Upgraded in 9f8df81.